kolibri/plugins/coach/assets/src/modules/lessonResources/handlers.js

```diff
@@ -267,18 +267,22 @@ function _prepLessonContentPreview(store, classId, lessonId, contentId) {
     getParams: { no_available_filtering: true },
   }).then(
     contentNode => {
-      const assessmentMetadata = contentNode.assessmentmetadata;
```

Comment:
Hrm, yes, I think this was a result of the refactor that removed the assessment state mapper, which presumably returned an empty object when it wasn't an exercise.